yamllint/cli.py

```diff
@@ -96,6 +114,9 @@ def show_problems(problems, file, args_format, no_warn):
             continue
         if args_format == 'parsable':
             print(Format.parsable(problem, file))
+        elif args_format == 'github' or \
+                (args_format == 'auto' and is_github_actions()):
```

Comment:
Can we drop `is_github_actions()` and simplify to:
```suggestion
        elif args_format == 'github' or (args_format == 'auto' and 
            'GITHUB_ACTIONS' in os.environ and 'GITHUB_WORKFLOW' in os.environ):
```
?